util/db/dbutil.go

```diff
@@ -325,13 +339,25 @@ func (db *Accessor) atomic(fn idemFn, commitLocker sync.Locker, extras ...interf
 		commitLocker.Unlock()
 	}
 
-	end := time.Now()
-	delta := end.Sub(start)
-	if delta > time.Second {
-		db.getDecoratedLogger(fn, extras).Warnf("dbatomic: tx took %v", delta)
-	} else if delta > time.Millisecond {
```

Comment:
Removed because?...

Reply:
The printing in debug wasn't really useful. It was just spitting it in *some* unit tests, and we generally ignored it completely. As some of the queries getting longer than 1 millisecond consistently, we always were outputting this, which again, became pointless.
I'm hoping that with this change in place, we'll be able to craft more precise warn timeouts.